geth/jail/jail_cell_test.go

```diff
@@ -109,3 +107,190 @@ func (s *JailTestSuite) TestJailLoopInCall() {
 		require.Fail("Failed to received event response")
 	}
 }
+
+func (s *JailTestSuite) TestJailLoopRace() {
+	require := s.Require()
+	require.NotNil(s.jail)
```

Comment:
I was getting rid of this code in the previous refactoring: it's not the responsibility of tests themselves to check that. If this happens, which is nearly impossible, it will be caught later anyway.

Reply:
Oh right. I added it originially before your refactor, and then cherry-picked and missed this.